### PR TITLE
copied and pasted twosum

### PR DESCRIPTION
**Pull request by hamsurtado**

### CODE REVIEW

File: functions/two_sum.py
Comment (kimcodes3) [inline functions/two_sum.py:1]: This is great! Please rename the function to two_sum to fit style guidelines

File: tests/two_sum_test.py
Comment (kimcodes3) [inline tests/two_sum_test.py:1]: All these cases are perfect for testing, please add an edge case to ensure functionality!

**Review by kimcodes3 — CHANGES_REQUESTED** [review]

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,7 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
+            if n[j] == t - n[i]:
+                return [i, j]
+            
+
```
Comment (red-med) [inline functions/two_sum.py:7]: looks good! yay

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+def test_returns_0_1_for_3_3_with_target_6():
+    # Arrange
+    n = [3, 3]
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```
Comment (red-med) [inline tests/two_sum_test.py:37]: maybe add a edge test case where none of the elements reach the target!

**Review by red-med — CHANGES_REQUESTED** [review]
looks good!

File: functions/two_sum.py
Comment (hamsurtado) [reply functions/two_sum.py:1]: I shall do this!

File: tests/two_sum_test.py
Comment (hamsurtado) [reply tests/two_sum_test.py:1]: Consider it done!
